config.py

```diff
@@ -1069,16 +1095,28 @@ def load_check_from_places(check_config, check_name, checks_places, agentConfig)
     load_success, load_failure = {}, {}
     for check_path_builder in checks_places:
         check_path, manifest_path = check_path_builder(check_name)
+
+        is_wheel = bool(not check_path and not manifest_path)
         # The windows SDK function will return None,
         # so the loop should also continue if there is no path.
-        if not (check_path and os.path.exists(check_path)):
+        if not (check_path and os.path.exists(check_path)) and not is_wheel:
             continue
 
-        check_is_valid, check_class, load_failure = get_valid_check_class(check_name, check_path)
+        prev_failures = bool(load_failure)
+        check_is_valid, check_class, load_failure = get_valid_check_class(check_name, check_path, from_site=is_wheel)
         if not check_is_valid:
+            load_error = load_failure.get(check_name, {}).get('error')
+            if is_wheel and not prev_failures and isinstance(load_error, ImportError):
+                load_failure = {}
             continue
 
-        if manifest_path:
+        if manifest_path or is_wheel:
```

Comment:
if `manifest_path` is false, we still call `validate_sdk_check(manifest_path)` and produce a warning, is this expected?

Reply:
the idea was to ship the manifest with the wheels, but it's not that straight-forward...

Reply:
If we delegate dependency management to pip (and assume an incompatible check can't just be installed) there's no need to ship/use the manifest at runtime